pkg/elasticsearch/dtos/back_up.go

```diff
@@ -0,0 +1,47 @@
+package dtos
```

Comment:
Nested dto structs are not supported. Do you test it with the frontend? And please only require minimal user data.